cli/installer/test-installer-containers.ps1

```diff
@@ -35,7 +36,15 @@ docker build  . `
         continue
     }
 
-    docker run -t azd-test
+    docker run $AdditionalRunArgs -t azd-test
+    if ($LASTEXITCODE) {
+        Write-Error "Validation run failed for $dockerfile"
+        $exitCode = 1
+
+        # Build failed, don't execute the container becuase we'll be executing
+        # the last successfully built container with the name azd-test
+        continue
```

Comment:
Does this comment make much sense here, given we're past execution?

I think we can simply say "Set overall installation failure, continue with next container to test" or something like that